### PR TITLE
Daniel

### PR DESCRIPTION
**Pull request by deplummer**

### CODE REVIEW

**Comment by mgoshorn** [comment]
@deplummer Need better commit names and PR names here Daniel. The commit names should give me an overview of changes for that commit.  The PR should have a name and description that tells me about the feature being integrated.

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (mgoshorn) [inline package-lock.json]: Why is there a package-lock.json file in a Spring project?  This should not be here.

File: pom.xml [hunk 2]
```diff
@@ -146,4 +146,4 @@
 
 	</dependencies>
 
-</project>
```
Comment (mgoshorn) [inline pom.xml:149]: Files should end with a line break. This change is a regression.

File: src/main/resources/META-INF/additional-spring-configuration-metadata.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{"properties": [
```
Comment (mgoshorn) [inline src/main/resources/META-INF/additional-spring-configuration-metadata.json:1]: What does this do?  Why do we need it?  What problem is it fixing?

**Review by mgoshorn — CHANGES_REQUESTED** [review]

**Comment by mgoshorn** [comment]
@deplummer Remove package-lock.json. This should not be a part of a spring app

File: package-lock.json (hunk 1 above)
Comment (deplummer) [reply package-lock.json]: Hi Mitch, somehow my file got corrupted.  I removed the package-lock.json file and updated the commit name.

File: src/main/resources/META-INF/additional-spring-configuration-metadata.json (hunk 3 above)
Comment (deplummer) [reply src/main/resources/META-INF/additional-spring-configuration-metadata.json:1]: The file got corrupted.

**Pull request merged**